### ISSUE TITLE
[Bug] Outputted type definitions are referencing the wrong path

### ISSUE THREAD

**Issue by tanzoniteblack** [colbyfayock/next-cloudinary#106]
# **Bug Report**

## **Describe the bug**

`dist/index.d.ts` is attempting to import the directory instead of the individual file, which is causing projects using the package to not correctly be able to utilize the types.

```js
export { default as CldImage } from './components/CldImage';
export { default as CldOgImage } from './components/CldOgImage';
```

I think this should be:

```js
export { default as CldImage } from './components/CldImage/CldImage';
export { default as CldOgImage } from './components/CldOgImage/CldOgImage';
```

## **Is this a regression?**

New bug based on my work to add typescript.

## **Expected behaviour**

When used as a dependency, projects should correctly see types for all the components.

### **Your environment**

N/A

## **Additional context**

I'll go ahead and work on this, should be pretty simple to fix.

**Comment by colbyfayock** [comment]
thanks @tanzoniteblack

**Comment by tanzoniteblack** [comment]
Update: This seemed like it should be easy, but I'm still struggling with it.

**Comment by colbyfayock** [comment]
hey @tanzoniteblack wanted to give you a heads up in case you planned to do any other tweaks to TS-related stuff on this project

i published via #111 a separate package to maintain the loader and utility functions as my plan is to be able to use that in other frameworks

https://github.com/colbyfayock/cloudinary-util

feel free to poke around. i converted everything to typescript and i think i got most htings taken care of. probably some things to still improve on there and here but progress :)

thanks for your help
